README.md

```diff
@@ -74,106 +98,37 @@ And here is the equivalent implementation in Jsonnet with
 }
 ```
 
-Using the Jsonnet command line, we can easily generate a
-`deployment.json` file from the above, which can then be sent to the
-cluster directly by `kubectl`:
+Save the file as `helloworld.libsonnet`, then run:
 
 ```bash
-$ jsonnet hello.jsonnet -m .         # Generates `deployment.json`.
-$ kubectl create -f deployment.json  # Dispatch to run on cluster.
+jsonnet helloword.libsonnet
 ```
 
-This is nice, but sometimes the default `Deployment` object is not
-precisely what you want.
+This command creates the `deployment.json` file that the 
+**ksonnet** snippet defines.
 
-A core goal of `kube.libsonnet` is to maintain the flexibility and
-expressiveness of the original Kubernetes API objects. To give you
-some idea of how easy it is to modify these objects, let's
-change the deployment specification to use a rolling update strategy 
-and a custom selector (see also [source][v2hellojsonnet]). We use
-`kube.libsonnet`'s mixins to make the changes.
+You can now apply this deployment to your Kubernetes cluster
+by running the following command:
 
-```c++
-// hello.jsonnet; imports omitted
-{
-  local nginxContainer =
-    container.Default("nginx", "nginx:1.7.9") +
-    container.NamedPort("http", 80),
-
-  "deployment.json":
-    deployment.FromContainer("nginx-deployment", 2, nginxContainer) +
-    deployment.mixin.spec.RollingUpdateStrategy() +
-    deployment.mixin.spec.Selector({ "app": "nginx" }),
-}
+```bash
+kubectl apply -f deployment.json
```

Comment:
Right after this example I would deep link to something in the examples directory, something like "Check out _this_ example of a cockroachdb chart written with kjsonnet." Then on the example add a link to the original chart, so that a user can easily open them both up in a tab and see the differences.